ehb_service/apps/api/views/externalsystem.py

```diff
@@ -98,10 +99,8 @@ def process(es, response):
         return self._read_and_action(request, process, pk)
 
 
-class ExternalSystemQuery(Resource):
-
-    supported_accept_types = ['application/json']
-    model = 'core.models.identities.ExternalSystem'
+@permission_classes((permissions.AllowAny,))
```

Comment:
Removed the @permissionsAllowAny header from all the classes. Created a api/authentication.py file to override Django authentication so that we can continue authenticating using the api tokens that we've created instead of switching over to Django-generated auth-tokens. 803fea7